daisy/step_delete_resources.go

```diff
@@ -364,7 +370,22 @@ func (d *DeleteResources) run(ctx context.Context, s *Step) DError {
 		return ret
 	}
 
-	// Delete networks after subnetworks have been deleted
+	// Delete firewalls after subnetworks have been deleted
```

Comment:
Any reason to wait for subnetworks to be deleted before deleting firewalls? It might not matter but deleting firewalls before deleting networks/subnets seems more natural.

Reply:
Good catch. If the subnetwork/network use the firewall. should be delete firewall first.